packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardBack.tsx

```diff
@@ -13,11 +13,11 @@ type TJurisdictionCardBackProps = {
     verificationDocs?: TJurisdictionCardItemVerification;
 };
 
-const JurisdictionCardBack: FC<TJurisdictionCardBackProps> = ({ setIsFlipped, verificationDocs }) => {
+const JurisdictionCardBack = ({ setIsFlipped, verificationDocs }: TJurisdictionCardBackProps) => {
     const verificationContents = jurisdictionVerificationContents();
-    if (!verificationDocs) return <></>;
+    if (!verificationDocs) return null;
     return (
-        <div className='absolute flex flex-col h-full transform backface-hidden rotate-y-180 gap-800 pt-[15px] px-800 pb-12000'>
+        <div className='absolute flex flex-col h-full transform backface-hidden [transform:rotateY(180deg)] gap-800 pt-[15px] px-800 pb-12000'>
```

Comment:
```suggestion
        <div className='absolute flex flex-col h-full backface-hidden [transform:rotateY(180deg)] gap-800 pt-[15px] px-800 pb-12000'>
```
if we are changing the style shouldn't we remove `transform`

Reply:
Updated in the Jurisdiction Modal PR.

Reply:
can we try this instead
```suggestion
        <div className='absolute flex flex-col h-full transform backface-hidden transform-y-180 gap-800 pt-[15px] px-800 pb-12000'>
```

Reply:
Already mentioned in https://github.com/binary-com/deriv-app/pull/12888

Reply:
I don't think tailwind supports 3d transforms, for that we would this additional library https://github.com/sambauers/tailwindcss-3d